style.css

```diff
@@ -88,91 +97,586 @@ body {
   height: 150px;
   font-weight: 700;
   font-size: 40px;
-  padding: 90px 12px 170px;
+  padding: 90px 24px 170px;
   line-height: 52px;
   color: #172b4d;
-  flex: none;
-  order: 0;
-  flex-grow: 0;
 }
 
 .container .bio {
   width: 327px;
   height: 146px;
-  left: 0;
-  top: 0;
-  padding: 3px 12px 24px;
-  font-style: normal;
+  padding: 3px 24px 24px;
   font-weight: 400;
   font-size: 16px;
   line-height: 24px;
+  color: #344563;
+  order: 1;
+}
+
+.container .lets_contact {
+  width: 157px;
+  height: 24px;
+  margin-top: 40px;
+  padding-left: 24px;
+}
+
+.container .lets_contact .contact {
+  font-weight: bold;
+  font-size: 16px;
+  line-height: 24px;
+  color: #7f8cff;
+}
+
+.social {
+  padding-top: 5%;
+  padding-bottom: 6%;
+}
+
+.social-icons-set {
+  display: flex;
+  padding: 2px;
+}
+
+.social-icon {
+  list-style: none;
+  margin: 0 10px;
+}
+
+.social-icon :hover {
+  color: #6070ff;
+}
+
+.social-icon a i {
+  font-size: 20px;
+  color: #505f79;
+  margin: 2px 12px;
+}
+
+.social-icon + .social-icon {
+  padding-left: 1px;
+}
+
+.deatails {
+  margin-top: 88px;
+  display: grid;
+  padding: 16px;
+  gap: 12px;
+  width: 375px;
+  height: 552px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+}
+
+.deatails2 {
+  margin-top: 88px;
+  display: grid;
+  padding: 16px;
+  gap: 12px;
+  width: 375px;
+  height: 596px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+}
```

Comment:
- [ ] As the project required, the section of the projects should be styled with CSS Grid, while each card should be styled with Flexbox. Kindly change the styling of each card to meet the project requirements.
 - 
<img width="1013" alt="image" src="https://user-images.githubusercontent.com/61919617/228617153-b7fc4f36-f4fc-49d3-b1e7-0f10f7560035.png">

 - [ ] The cards has a good style similar to the template, but it has no margins horizontally, making it on the limit of the screen. Kindly add some margins to make the card look like it is 'floating' like the Figma template.
 - Your design:
 - 
<img width="490" alt="image" src="https://user-images.githubusercontent.com/61919617/228617635-04913b51-ee7a-4921-b5eb-27c695ccfa66.png">

 - Figma template:
 - 
<img width="298" alt="image" src="https://user-images.githubusercontent.com/61919617/228617720-0918d775-4063-4e71-b682-3d8eae2e87ed.png">